test/ng/qSpec.js

```diff
@@ -2113,7 +2113,7 @@ describe('q', function() {
       });
 
 
-      it('should log exceptions throw in a progressack and stop propagation, but shoud NOT reject ' +
+      it('should log exceptions thrown in a progressback and stop propagation, but should NOT reject ' +
```

Comment:
So, upon trying a basic pair of commits to for this (see below), it became apparent that `src/ng/q.js` uses `errback` and `progressBack` as the names of the functions. I don't feel comfortable changing either name in a cleanup PR like this one, so I'm going to settle on `progressBack` and leave `errback` for now.

If someone wants to replace `errback` and `progressBack` with `errorCallback` and `progressCallback` respectively, they can do that in a subsequent PR.